libs/output-mapping/tests/Writer/StorageApiLocalTableWriterTest.php

```diff
@@ -2276,4 +2469,47 @@ private function getTableIdFromJobDetail(array $jobData): string
 
         self::fail(sprintf('Cannot detect tableId from %s job', $operationName));
     }
+
+    private function findTableCreateJobForTable(string $tableId, array $jobsIds): ?array
```

Comment:
JobIds jsou v poradi, v jakem Finder nasel soubory na disku. Ted tam neni sort a lokalne a na CI to jde v jinem poradi. Proto to musim vyhledavat.